src/include/daos/tests_lib.h

```diff
@@ -337,5 +337,18 @@ int dmg_storage_device_list(const char *dmg_config_file, int *ndisks,
  */
 int dmg_storage_set_nvme_fault(const char *dmg_config_file,
 			       char *host, const uuid_t uuid, int force);
-
+/**
+ * Get NVMe Device health stat.
+ *
+ * \param dmg_config_file
+ *		[IN]	DMG config file
+ * \param host	[IN]	Nvme set to faulty on host name provided. Only single
```

Comment:
"Host name of the server that the device to query resides?" doesn't need to be set to faulty necessarily to use this right?

Reply:
No, not required to set this faulty. This is general query and not depend on device stats.

Reply:
I would remove the "faulty" part of the param definition then

Reply:
Ah got it, It was copy paste. I will change it.